axon-mongo-spring-boot-autoconfigure/src/main/java/org/axonframework/extensions/mongo/springboot/AxonMongoProperties.java

```diff
@@ -0,0 +1,203 @@
+     */
+    public EventStore getEventStore() {
+        return eventStore;
+    }
+
+    /**
+     * Defines the AutoConfiguration settings for the event store.
+     *
+     * @param eventStore the AutoConfiguration settings for the event store.
+     */
+    public void setEventStore(EventStore eventStore) {
+        this.eventStore = eventStore;
+    }
+
+    /**
+     * Retrieves the AutoConfiguration settings for the saga store.
+     *
+     * @return the AutoConfiguration settings for the saga store
+     */
+    public SagaStore getSagaStore() {
+        return sagaStore;
+    }
+
+    /**
+     * Defines the AutoConfiguration settings for the saga store.
+     *
+     * @param sagaStore the AutoConfiguration settings for the saga store.
+     */
+    public void setSagaStore(SagaStore sagaStore) {
+        this.sagaStore = sagaStore;
+    }
+
+    public static class TokenStore {
+
+        /**
+         * Enables creation of the {@link org.axonframework.extensions.mongo.eventsourcing.tokenstore.MongoTokenStore}
+         */
+        private boolean enabled = true;
+
+        /**
+         * Indicates whether creating the
+         * {@link org.axonframework.extensions.mongo.eventsourcing.tokenstore.MongoTokenStore} is enabled.
+         *
+         * @return true if creating the token store is enabled, false if otherwise
+         */
+        public boolean isEnabled() {
+            return enabled;
+        }
+
+        /**
+         * Enables (if {@code true}, default) or disables (if {@code false}) creating the
+         * {@link org.axonframework.extensions.mongo.eventsourcing.tokenstore.MongoTokenStore}.
+         *
+         * @param enabled whether to enable token store creation
+         */
+        public void setEnabled(boolean enabled) {
+            this.enabled = enabled;
+        }
+    }
+
+    public static class EventStore {
+
+        /**
+         * Enables creation of the
+         * {@link org.axonframework.extensions.mongo.eventsourcing.eventstore.MongoEventStorageEngine}
+         */
+        private boolean enabled = false;
+
+        /**
+         * Indicates whether creating the
+         * {@link org.axonframework.extensions.mongo.eventsourcing.eventstore.MongoEventStorageEngine} is enabled.
+         *
+         * @return true if creating the event store is enabled, false if otherwise
+         */
+        public boolean isEnabled() {
+            return enabled;
+        }
+
+        /**
+         * Disables (if {@code false}, default) or enables (if {@code true}) creating the
+         * {@link org.axonframework.extensions.mongo.eventsourcing.eventstore.MongoEventStorageEngine}.
+         *
+         * @param enabled whether to enable event store creation
+         */
+        public void setEnabled(boolean enabled) {
+            this.enabled = enabled;
+        }
+    }
+
+    public static class SagaStore {
+
+        /**
+         * Enables creation of the
```

Comment:
I think it's good to state the default here. That will show up in user's IDE in a `*.properties` file.

Reply:
Added